pkg/network/kube_proxy_test.go

```diff
@@ -98,6 +127,66 @@ winkernel:
 `))
 }
 
+func TestKubeProxyIPv6Config(t *testing.T) {
+	g := NewGomegaWithT(t)
+
+	errs := validateKubeProxy(&configIPv6)
+	g.Expect(errs).To(HaveLen(0))
+
+	cfg, err := kubeProxyConfiguration(map[string]operv1.ProxyArgumentList{
+		// special address+port combo
+		"metrics-bind-address":   {"fd00:1234::4"},
+		"metrics-port":           {"999"},
```

Comment:
use a port above 32767 here to test that fix?

Reply:
done